docs/concepts.rst

```diff
@@ -759,25 +759,22 @@ For example, consider the following dag:
   from airflow.utils.trigger_rule import TriggerRule
 
 
-  dag = DAG(
-      dag_id='latest_only_with_trigger',
-      schedule_interval=dt.timedelta(hours=4),
-      start_date=dt.datetime(2016, 9, 20),
-  )
+  with DAG(dag_id='latest_only_with_trigger',
+           schedule_interval=dt.timedelta(hours=4),
+           start_date=dt.datetime(2016, 9, 20)) as dag:
```

Comment:
Do we need the `as dag` if we don't reference it inside the block anywhere? (I think this would cause a warning from flake8 if it this code was run through it?)

Reply:
No, we don't, good spot! I'll fix when I get the time (probably this weekend)

Reply:
All done